tests/model_meta/models.py

```diff
@@ -5,7 +5,9 @@
 
 
 class Relation(models.Model):
-    pass
+    content_type = models.ForeignKey(ContentType, models.CASCADE, related_name="+")
+    object_id = models.PositiveIntegerField()
+    content_object = GenericForeignKey()
```

Comment:
As above, fields required to fill in for `GenericRelation`s below, preventing `contenttypes.E004` errors.